docs/source/workshop.rst

```diff
@@ -478,21 +478,95 @@ Conferences as large as ICLR 2019 will often have a number of reviews that excee
+
+We now obtain the Original Submissions. In this case, we also place the Original Submissions in a dictionary (original_notes) where the key is the id of the Submission.
+
+	>>> original_notes = {note.id: note for note in openreview.tools.iterget_notes(client, invitation = 'auai.org/UAI/2019/Conference/-/Submission')}
+
+Finally, we iterate over all the Decision notes from the venue and filter by decision. Each Decision note contains a forum id that points to the Blind Submission. Similarly the Blind Submission will point to the Original Submission.
+
+	>>> accepted_submissions = []
+	>>> for decision_note in all_notes:
+	...	if 'Accept' in decision_note.content['decision']:
+	...		# Get Blind Submission using the forum id
+	...		blind_note = blind_notes[decision_note.forum]
+	...		# Get the Original Submission using the blind_note 'original' field
+	...		accepted_submissions.append(original_notes[blind_note.original])
+
+* Reviewers' information is visible
+
+Since we are interested in the Submission whose Decision contains 'Accept', we first get all the Decision notes for the venue we are interested in. We first obtain all the Decision notes for a particular venue. This step is the same regardless of the format of the venue.
+
+	>>> all_notes = openreview.tools.iterget_notes(client, invitation = 'MIDL.io/2019/Conference/-/Paper.*/Decision')
+
+In some cases, however, the invitation may have a different format (note the position of the dash '-'):
+
+	>>> all_notes = openreview.tools.iterget_notes(client, invitation = 'roboticsfoundation.org/RSS/2019/Workshop/Robust_Autonomy/Paper.*/-/Decision')
+
+We now obtain the Original Submissions. In this case, we also place the Original Submissions in a dictionary (original_notes) where the key is the id of the Submission.
+
+	>>> original_notes = {note.id: note for note in openreview.tools.iterget_notes(client, invitation = 'MIDL.io/2019/Conference/-/Full_Submission')}
 
 We then filter the Decision notes that were accepted and use their forum ID to get the corresponding Submission.
 
-	>>> accepted_submissions = [id_to_submission[note.forum] for note in all_decision_notes if note.content['decision'] == 'Accept']
+	>>> accepted_submissions = [original_notes[note.forum] for note in all_notes if note.content['decision'] == 'Accept']
+
+Retrieving all the author names and e-mails from accepted Submissions
+---------------------------------------------------------------------
+Our goal is to retrieve the information of the authors (e-mails and/or names). This information is stored in the Original Submission, which is a note. However, depending on the venue, this information might not be directly visible and some 'tricks' may be needed. Therefore, the first thing you need to do is determine if the Submissions you are interested in are double blind or not. If the authors' information is not visible to the reviewers and viceversa, then the venue is double blind.
```

Comment:
This is the same that the previous example, we can skip it and show how to iterate through the accepted papers to get the author names and emails.